composer.json

```diff
@@ -21,20 +21,21 @@
 		}
 	],
 	"require": {
-		"php": "^7.4 || ^8.0",
+		"php": "^8.1",
 		"ext-json": "*",
-		"nikic/php-parser": "^4.10.4",
-		"typo3/cms-backend": "^11.4",
-		"typo3/cms-core": "^11.4"
+		"nikic/php-parser": "^4.15.4",
+		"typo3/cms-backend": "^12.2.0 || 12.3.x-dev || dev-main",
```

Comment:
this looks outdated

Reply:
Strange. This must be very old and not have been updated for some months 😅

Reply:
there are newer commits with that file ...

Reply:
I'd propose just merge it in a new branch and then we can check easier.
A large stack like this is hard to oversee.

Reply:
You dont need to read every commit. This PR is for the whole branch. There is already a seperate branch for this. 
Just look at the changed files as these lines are already updated. It was a little bit confusing.

Reply:
Go for it, we need to put the things together we have till now before proceeding.